pkg/appgw/backendhttpsettings_test.go

```diff
@@ -154,7 +164,7 @@ var _ = Describe("Test the creation of Backend http settings from Ingress defini
 
 			for _, setting := range httpSettings {
 				if *setting.Name == DefaultBackendHTTPSettingsName {
-					Expect(int32(80)).To(Equal(*setting.Port), "default backend port %d should be 80", *setting.Port)
+					Expect(int32(443)).To(Equal(*setting.Port), "default backend port %d should be 80", *setting.Port)
```

Comment:
`Expect(int32(443)).To(Equal(*setting.Port), "default backend port %d should be 443", *setting.Port)` update the string with new value